README.rst

```diff
@@ -1,6 +1,5 @@
 |build-status|
 
-A Gnus backend for Discourse.
```

Comment:
It appears when you `make README.rst`, the logic to extract commentary from `nndiscourse.el` got lost.